lib/app/redirectback.coffee

```diff
@@ -2,10 +2,12 @@
 # Redirects back based on query params, session, or w/e else.
 # Code stolen from Force, thanks @dzucconi!
 #
+opts = require '../options'
 sanitizeRedirect = require './sanitize_redirect'
 
 module.exports = (req, res) ->
   url = sanitizeRedirect(
+    (opts.afterSignupPagePath if req.artsyPassportSignedUp) or
```

Comment:
Putting this as the first condition essentially means onboarding can't be skipped since this takes precedence over other `redirect-to` params, intentional?